server/sonar-server/src/main/java/org/sonar/server/computation/ws/IsQueueEmptyWs.java

```diff
@@ -60,6 +60,8 @@ public void define(WebService.NewController controller) {
       controller
         .createAction("is_queue_empty")
         .setDescription("Check if the queue of Compute Engine is empty")
+        .setResponseExample(getClass().getResource("is_queue_empty-example.txt"))
```

Comment:
As is_queue_empty-example.txt contains only "true", `.setResponseExample("true")` is simpler.